test/libengine/engine_bind_test.cc

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
As I mentioned in my PR comment, it's more idiomatic for Google to have tests and src in the same directory,